docker/testing.Dockerfile

```diff
@@ -36,9 +36,6 @@ ENV PATH "$PATH:~/.composer/vendor/bin"
 # Configure php
 RUN echo "date.timezone = UTC" >> /usr/local/etc/php/php.ini
 
-# Remove exec statement from base entrypoint script.
-RUN sed -i '$d' /usr/local/bin/app-entrypoint.sh
```

Comment:
This tells me we want everything the setup part of the script does to get WP running.